neuromation/cli/main.py

```diff
@@ -178,6 +178,16 @@ def print_options(
     is_flag=True,
     help="Trace sent HTTP requests and received replies to stderr.",
 )
+@click.option(
+    "--hide-token/--no-hide-token",
```

Comment:
No, it can be `reveal-token`, `demonstrate-token`,  `unfold-token`, `expose-token`,  `present-token`, `unhide-token` at the end. In boolean logic of `click`, the opposite to `--hide-token` is `--no-hide-token`, thus the user can easily understand what is the default option `True`.

Note also that `--no-no-hide-token` is an invalid option:
```
$ n --no-no-hide-token --trace ls
Usage: neuro [OPTIONS] COMMAND [ARGS]...
Try "neuro --help" for help.

Error: no such option: --no-no-hide-token
```

> The default is never shown. <...> Even None as not set is appropriate.

Thank you for this comment, I added the option `show_default=True` and changed the default `default=True` without changing the logic.

Reply:
> I added the option show_default=True and changed the default default=True without changing the logic

This approach was erroneous. I solved this problem in the same way as it's solved for `"--http-auth/--no-http-auth"`: `is_flag=True`, `default=None` and the default value is manually written to the help message.

Reply:
ok